### PR TITLE
Shopping Cart Impl

### PR DESCRIPTION
**Pull request by KDanil2000**

### CODE REVIEW

**Review by TetianaDenchyk — APPROVED** [review]

File: src/main/java/model/Book.java [hunk 1]
```diff
@@ -43,6 +47,12 @@ public class Book {
             inverseJoinColumns = @JoinColumn(name = "category_id")
     )
     private Set<Category> categories = new HashSet<>();
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @OneToMany(cascade = CascadeType.REMOVE, mappedBy = "book")
```
Comment (Elena-Bruyako) [inline src/main/java/model/Book.java]:
```suggestion
    @OneToMany(cascade = CascadeType.REMOVE, mappedBy = "book", orphanRemoval = true)
```

File: src/main/java/repository/CartItemRepository.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import java.util.Optional;
+import model.CartItem;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CartItemRepository extends JpaRepository<CartItem, Long> {
+    Optional<CartItem> findByIdAndShoppingCartId(Long cartItemId, Long shoppingCartId);
+
+
```
Comment (Elena-Bruyako) [inline src/main/java/repository/CartItemRepository.java]: Remove empty lines

File: src/main/resources/db/changelog/changes/09-create-table-cart-items.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: quantity
+                  type: int
+                  constraints:
+                    nullable: false
```
Comment (Elena-Bruyako) [inline src/main/resources/db/changelog/changes/09-create-table-cart-items.yaml:30]: add empty line at the end for each changelogs

**Review by Elena-Bruyako — APPROVED** [review]
Good job! 
Approve but left some comments.

**Pull request merged**